picard/ui/searchdialog.py

```diff
@@ -594,27 +666,38 @@ def handle_reply(self, document, http, error):
         self.fetch_coverarts()
```

Comment:
Fixed this by changing it to         
```
QTimer().singleShot(2000, self.fetch_coverarts)
```
The problem is that redering the cells takes time. So they are not visible immediately after the call to handle_reply.
Although this is a bit hacky, a better way would be to connec to the cell's show event. But I guess you get what the problem is.

Reply:
It was my guess, i'll try to find a good way to do it. Thanks ;)

Reply:
Fixed in 766c31a